src/views/product-root-docs-path-landing/components/icon-card-link-grid/icon-card-link-grid.module.css

```diff
@@ -0,0 +1,10 @@
+/* @TODO make responsive */
```

Comment:
🗒️  Going to do this after I finish writing up this PR!

Reply:
⛏️ Nitpick: should we handle this before merging? (IMO okay to handle in follow-up since Vault's current single `API` icon card doesn't break here despite still being 3 columns on mobile, and ditto with Waypoint's content)

Reply:
Already did it! I was just about to comment with the commit number. 😊
https://github.com/hashicorp/dev-portal/pull/452/commits/97b10a5f9965ca8f6f224727cc1cf6769a2e7d89

Reply:
oh nice, i think i had the review tab open a while and didn't refresh! 😅